README.md

```diff
@@ -1,150 +1,235 @@
-
-The fallback microservice B will provide older information about the space station incase the stations on board computer fails so that a connection can still be made to the station rather than space ships not being able to comunicate with the station at all.
-
-### Prerequisites
-1. Maven (https://maven.apache.org/install.html) - for building, testing and running our microservices
-2. npm (https://www.npmjs.com/get-npm) - for installing Angular and the Angular CLI
-3. Angular (https://www.npmjs.com/package/angular) - for building and running our web application
-4. Angular CLI (https://cli.angular.io/)
-4. Git CLI (https://git-scm.com/downloads) - for downloading the source files for this tutorial
-5. A Java IDE (Eclipse, Visual Studio Code, Netbeans etc) - for coding our tutorial
-6. Connection to the Internet - to pull down the required dependancies
-
-
-### To Note!
-
-We understand that normally you would have a separate repository for each microservice but to make this tutorial easier to consume we have decided to keep all the source in one repository to save the user having to download multiple repositories.
-
-We are planning to extent this tutorial to use Docker, Kuberneties and other technologies but we wanted to limit the prerequisites so that users can get up and ready quickly.
-
-## Information About The Files We Have Provided
-We have provided some file for you to make this tutorial easier to consume that I will talk about here.
-Firstly we have provided you with the basic directory structure for all the microservices.
-Then we have provided you with a pom.xml file that is used by maven to build your microservices and download all the required dependancies. In this file you can set what the application is named `<app.name>LibertyMicroServiceOne-1.0</app.name>`, the ports assigned to Liberty 
-```
-<testServerHttpPort>9090</testServerHttpPort> <testServerHttpsPort>9444</testServerHttpsPort>
-```
-the dependancies you require like microProfile 1.2
-```
-<dependency>
-    <groupId>org.eclipse.microprofile</groupId>
-    <artifactId>microprofile</artifactId>
-    <version>1.2</version>
-    <scope>provided</scope>
-    <type>pom</type>
-</dependency>
-```
-and the applicaiton server you are using so in our case open Liberty
+###About this module
+
+This module is a continuation of module 3 however, the finished module 3 code has been provided in this module 4 branch.
+
+This module will demonstrate MicroProfile Config. MP Config is a solution to externalise configuration from microservices, enabling users to separate configuration from code to enable microservices to be successfully run in different environments requiring different configurations. This helps to ensure the high portability of microservices.
+
+1.  Firstly, the static configuration properties in a static configuration properties file need to be set (this could be one of three pre-existing config files).
```

Comment:
Formatting here does not look correct. You just need one space between the 1. and the word otherwise it will all go into one paragraph rather than on a new line.